jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/EtcdClusterFactory.java

```diff
@@ -21,10 +21,15 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.net.URI;
 import java.util.List;
+import java.util.Set;
```

Comment:
is this being used? the ci complains this is unused!

Reply:
It is not. Removed.